content/develop/_index.md

```diff
@@ -10,35 +10,99 @@ tags = [ "theme" ]
+3. Components should be context aware, if possible
     - `Content` components should be equipped with styles for the `on="dark"` attribute, so that they can be used on both light and dark backgrounds. 
 4. Framework agnostic
-    - A RHElement should “just work” when you drop it onto any page (provided the proper polyfills are there). It should have ALL the styles it needs, such as font-family properties.
+    - A PatternFly Element should "just work" when you drop it onto any page (provided the proper polyfills are there). It should have ALL the styles it needs, when coupled with a PFE Theme.
 
-## Container RHElements
+## Container Elements
 
-A container never imposes padding or styles on the children. It is only concerned with their horizontal or vertical alignment and spacing. For example in a card, you might apply margins to the slots, but you would not apply padding or color not to the content inside of slots.
+A container never directly styles the children inside it. It is built to manage only their horizontal or vertical alignment and spacing. For example, in a card, you might apply margins around the slots, but you would not apply padding or color to the content inside of slots.
 
-**Example: [rh-card](https://github.com/RHElements/rhelements/blob/master/elements/rh-card/src/rh-card.scss)**
+**Example: [Card](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-card/src)**
 
-This component has background color options, and attributes to change the vertical spacing between the slots.
+This component has background color options and attributes to change the vertical spacing between the slots but does not have any influence on the general styles or typography of its children.
 
-## Content RHElements
+
+## Content Elements
 
 A component always touches all four sides of its parent container. For example, if you are creating a teaser component, it should not have padding or margins around the type. It might, however have margins between the headline and author.
 
-**Example: [rh-cta](https://github.com/RHElements/rhelements/blob/master/elements/rh-cta/src/rh-cta.scss)**
+**Example: [Call-to-action](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-cta/src)**
+
+This component has attributes which invoke the multiple style changes on a call-to-action link. It also has `on="dark"` styles built-in so that colors will automatically adjust on dark surfaces
+
+## Combo Elements
+
+These elements are designed to create easy-to-use combinations of other smaller components. Ideally they also bundle up attributes into a few configurations.
+
+**Example: [Hide-show](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-hide-show)**
+
+
+## Opt-in Features
+- General features would be opt-in except for those that are declared by the design system. If the design system calls for previous and next buttons, you can opt-out but the default should match the suggested use.
+
```

Comment:
### Examples:

This markup would render the pagination component, with the jump link feature:
```
<pfe-pagination pfe-include-jump-link>
```

This markup would render the pagination component, without the jump link feature:
```
<pfe-pagination pfe-include-jump-link="false">
```

If, for some reason, styles are attached to the attribute which enables/disables features, use this CSS syntax (omit the `="true"` value.
```
[pfe-include-jump-link] {
    background: pink;
}
```

If it's necessary to "unset" styles, you will have to use the value of false.
```
[pfe-include-jump-link="false"] {
    border: blue solid 2px;
}
```

Reply:
Another approach would be:

> If there are any styles associated with a boolean attribute, the JavaScript should check for the attribute and if it exists, add a class to the ShadowDOM to which the styles can be attached.

Reply:
++ do we want to add that idea to the docs now, or wait on that?

Reply:
I'm okay either way.  Probably better to push this forward and we can always iterate.